content/en/marketplace/marketplace.md

```diff
@@ -0,0 +1,192 @@
+6. [Submit a pull-request](#6-submit-a-pull-request)
+7. [Approve tile for release](#7-approve-tile-for-release)
+
+### 1. Choose an integration type
+
+There are a few different ways to integrate with Datadog. Depending on the use case and type of integration, choose the approach that makes the most sense for your offering. 
+
+#### [**1. Datadog Agent-based Integration**](https://docs.datadoghq.com/developers/integrations/)
+
+##### [OpenMetrics check](https://docs.datadoghq.com/developers/prometheus/)
+* A OpenMetrics check is suitable for gathering telemetry data from existing applications that expose metrics using the Open Metrics standard.
+
+##### [Python Check](https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=configurationtemplate#write-the-check)
+* A Python check is suitable for monitoring services or products that do not expose metrics in a standard format. The Python check is used to collect telemetry data from various APIs or command line tools.
+
+##### [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent)
+* DogStatsD is suitable for applications that already emit telemetry using the StatsD protocol. Datadog adds additional Datadog-specific extensions to the StatsD protocol including:
+    * Histogram Metric Type
+    * Service Checks
+    * Events
+    * Tagging
+
+#### [**2. Datadog REST API Integration**](https://docs.datadoghq.com/api/)
+An API integration fits well for enriching and submitting data from your backend, or pulling data directly out of Datadog. API integrations also work well for building a connector between Datadog and another SaaS platform. 
+
+**Note:** that a Datadog API Key is required to submit data to a Datadog API endpoint, while an App Key is required for querying data from Datadog, or for creating resources within the Datadog App.
+
+#### **3. Tile Only Listing**
+For Marketplace offerings that include a standalone SaaS or services offering, with no exchange of data, only a tile is needed. The Development Toolkit offers a command option to create tile-only scaffolding: ddev create -t tile "<Offering Name>" 
+
+### 2. Build a bi-directional integration
+
+While pulling information from Datadog may also be useful, to be listed on the Datadog Integrations page or the Marketplace, the integration must be bi-directional. That is, it must also push data into Datadog. 
+
+Integrations can send the following data to Datadog:
+
+1. [Metrics](https://docs.datadoghq.com/api/latest/metrics/)
+2. [Logs](https://docs.datadoghq.com/api/)
+3. [Events](https://docs.datadoghq.com/api/latest/events/)
+4. [Service Checks](https://docs.datadoghq.com/api/latest/service-checks/)
+5. [Traces](https://docs.datadoghq.com/api/latest/tracing/)
+6. [Incidents](https://docs.datadoghq.com/api/latest/incidents/)
+7. [Security Events](https://docs.datadoghq.com/api/latest/security-monitoring/)
```

Comment:
I think we should denote somehow here which integration types support emitting which pieces. 
I.e. A Python check can't create an Incident or a Security Event.

Reply:
I agree this would be helpful to identify for partners. I'll ask on the agent-integrations channel and try to update the documentation.

Reply:
I am going to resolve for now, so we don't block, while I wait to hear back from Agent Integrations. Thanks